WcaOnRails/app/controllers/results_submission_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class ResultsSubmissionController < ApplicationController
+  require 'fileutils'
+  before_action :authenticate_user!
+
+  def edit
+    @competition = Competition.find(params[:competition_id])
+    redirect_to_root_unless_user(:can_submit_competition_results?, @competition) && return
+
+    @contact = WebsiteContact.new(your_email: "delegate@wca.com", to_email: "results@wca.com", subject: "Results")
+  end
+
+  def submit
+    @competition = Competition.find(params[:competition_id])
+    redirect_to_root_unless_user(:can_submit_competition_results?, @competition) && return
+
+    results_file = params[:results]
```

Comment:
Hmmm I tried that but for some reason flash.now doesn't render what I'm expecting:

![flashnow](https://user-images.githubusercontent.com/4351932/33687357-d4b5976a-da8c-11e7-84ef-9d1017fc149c.png)

Reply:
How does the line of code look like? Did you use `flash.now[:error]`?

Reply:
I used `:danger` instead of `:error` because `:error` didn't have any formatting and was rendering like this:

![flashnow](https://user-images.githubusercontent.com/4351932/33690992-ecbc0c82-da9a-11e7-8e39-d2fb486c9288.png)

Fixing the alignment is also on my list of things to do for this ticket